src/modules/business/services/businessRating/UpdateBusinessRatingService.ts

```diff
@@ -0,0 +1,42 @@
+import { injectable, inject } from 'tsyringe';
+
+import AppError from '@shared/infra/http/errors/AppError';
+
+import BusinessRating from '../../infra/typeorm/entities/BusinessRating';
+import IBusinessRatingRepository from '../../repositories/IBusinessRatingRepository';
+
+export interface IRequest {
+  value: number;
+  business_rating_id: string;
+}
+
+@injectable()
+export default class UpdateBusinessRatingService {
+  constructor(
+    @inject('BusinessRatingRepository')
+    private businessRatingRepository: IBusinessRatingRepository,
+  ) {}
+
+  public async execute({
+    business_rating_id,
+    value,
+  }: IRequest): Promise<BusinessRating> {
+    const businessRating = await this.businessRatingRepository.findById(
+      business_rating_id,
+    );
+
+    if (!businessRating) {
+      throw new AppError('Rating  not found.', 404);
```

Comment:
```suggestion
      throw new AppError('Rating not found.', 404);
```
there a double space here